scheduler/scheduler.go

```diff
@@ -245,7 +250,8 @@ func createResources(toCreate chan *ScheduledResource, created chan string, ccLi
 			}
 
 			for {
-				time.Sleep(time.Millisecond * 100)
+				log.Printf("Checking status for %s", r.Key())
+				time.Sleep(time.Millisecond * 1000)
```

Comment:
Good idea with informing about change, but I didn't want to introduce too much changes in this PR (it's already quite big as it is). Do you think it should be done in this PR?